db/schema.rb

```diff
@@ -392,10 +392,9 @@
     t.index ["ancestry"], name: "index_comments_on_ancestry"
     t.index ["ancestry"], name: "index_comments_on_ancestry_trgm", opclass: :gin_trgm_ops, using: :gin
     t.index ["commentable_id", "commentable_type"], name: "index_comments_on_commentable_id_and_commentable_type"
-    t.index ["commentable_type"], name: "index_comments_on_commentable_type"
     t.index ["created_at"], name: "index_comments_on_created_at"
-    t.index ["deleted"], name: "index_comments_on_deleted"
-    t.index ["hidden_by_commentable_user"], name: "index_comments_on_hidden_by_commentable_user"
+    t.index ["deleted"], name: "index_comments_on_deleted", where: "(deleted = false)"
+    t.index ["hidden_by_commentable_user"], name: "index_comments_on_hidden_by_commentable_user", where: "(hidden_by_commentable_user = false)"
```

Comment:
As it turns out, adding an index to a boolean column (especially one that's majority one value - `false`) doesn't do much at all for performance. Partial indexes have proven to be better in these cases so we're giving this a shot.